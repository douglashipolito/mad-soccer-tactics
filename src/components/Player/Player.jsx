import React from "react";
import "./Player.scss";

class Player extends React.Component {
  render() {
    return (
      <div className="player-component">
        <span className="player-name">{this.props.name}</span>
        <figure className="player-image-container">
          <img
            className="img-responsive player-image"
            alt={this.props.name}
            src={this.props.avatar}
          />
        </figure>
      </div>
    );
  }
}

export default Player;
