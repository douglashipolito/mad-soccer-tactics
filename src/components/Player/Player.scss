.player-component {
  $player-size: 7vw;
  width: $player-size;
  height: $player-size;

  .player-image-container {
    margin: 0;
    padding: 0;
    width: 100%;
    height: 100%;
    text-align: center;
  }

  .player-name {
    display: block;
    text-align: center;
    color: #fff;
  }
}
