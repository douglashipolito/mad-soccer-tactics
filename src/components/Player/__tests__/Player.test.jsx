import React from "react";
import { create } from "react-test-renderer";
import Player from "../";

test("Player Renders", () => {
  const string = create(<Player />);
  expect(string).toMatchSnapshot();
});
