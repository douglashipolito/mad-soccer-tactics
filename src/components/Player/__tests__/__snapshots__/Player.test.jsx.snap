// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Player Renders 1`] = `
<div
  className="player-component"
>
  <span
    className="player-name"
  />
  <figure
    className="player-image-container"
  >
    <img
      className="img-responsive player-image"
    />
  </figure>
</div>
`;
